staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1559,3 +1563,35 @@ message CheckpointContainerRequest {
 }
 
 message CheckpointContainerResponse {}
+
+message GetEventsRequest {
```

Comment:
nit: why is there a newline here? the others that are emtpy (like checkpoint above) are just {}